docs/adding-new-resource.md

```diff
@@ -0,0 +1,705 @@
+<!--
+SPDX-FileCopyrightText: 2023 The Crossplane Authors <https://crossplane.io>
+
+SPDX-License-Identifier: CC-BY-4.0
+-->
+
+### Prerequisites
+
+To follow this guide, you will need:
+
+1. Obtained a Kubernetes Cluster: For manual/local effort, generally a Kind
+cluster is sufficient and can be used. For detailed information about Kind see
+[this repo]. An alternative way to obtain a cluster is: [k3d]
+2. [Go 1.21+] installed and configured.
+3. [Terraform v1.5.5+] installed locally.
+4. [goimports] installed.
+
+# Adding a New Resource
+
+There are long and detailed guides showing [how to bootstrap a
+provider][provider-guide] and [how to configure resources][config-guide]. Here
+we will go over the steps that will take us to `v1beta1` quality.
+
+1. Fork the provider to which you will add resources and create a feature
+branch.
+
+2. Go to the Terraform Registry page of the resource you will add, we will add
```

Comment:
I understand we will expand why we need this in the next items. However, still we may consider adding one sentence to clarify the need of this opening the registry page item.